lib/driver.js

```diff
@@ -1075,10 +1079,131 @@ export class XCUITestDriver extends BaseDriver {
   }
 
   /**
-   *
-   * @param {string} cmd
-   * @param {...any} args
-   * @returns {Promise<any>}
+   * Checks if an error is related to WDA connection issues
```

Comment:
these all new method overcomplicate the driver class. consider extracting them to a separate module and make to mixins, similar to other commands